modules/serverlist.js

```diff
@@ -0,0 +1,35 @@
+const config = require("../config.json");
+const util = require("./util.js");
+
+var server_list;
+var client;
+
+exports.init = (cl) => {
+	client = cl;
+	util.init(cl);
+	util.fetchMessage(message => {
+		console.log(`Fetched server list message in #${message.channel.name}`);
+		server_list = message;
+		loop();
+	}, config.server_list);
+}
+
+
+
+function loop() {	
+	util.XMLHttpRequest(servers => {
+		
+		servers = util.sortServers(servers);
+
+		util.updateServerList(server_list, servers);
+
+		let total_players = servers.reduce((t, x) => t + x.currentPlayers, 0);
+		client.user.setPresence({ status: 'online', game: { name: `${total_players} in KAG | ${config.prefix}help` } });
```

Comment:
This shouldn't be in the server list module, but it does need a loop to update it.